python/xorbits/_mars/services/scheduling/worker/execution.py

```diff
@@ -44,6 +45,48 @@
 
 logger = logging.getLogger(__name__)
 
+
+class StatusMonitorActor(mo.Actor):
+    def __init__(self, **kw):
+        super().__init__()
+        self._records = dict()
+
+        if kw:  # pragma: no cover
+            pass
+
+    async def __post_create__(self):
+        pass
+
+    async def __pre_destroy__(self):
+        pass
+
```

Comment:
Do not need to define them.

Reply:
These useless codes are used as placeholders for future functionality and will not appear in the last commit.